docs/source/upgrade-guide.rst

```diff
@@ -3,6 +3,23 @@ Upgrade Guide
 
 This upgrade guide lists all breaking changes in plone.restapi and explains the necessary steps that are needed to upgrade to the lastest version.
 
+
```

Comment:
@deiferni we should add instructions here how to actually run the migration step (via ZMI). The alternative would be to provide a browser view that runs the upgrade step.

Reply:
@tisto ok, will amend the following:
- implement an upgrade that checks everything that `is_folderish` according to the catalog (currently i only have a function that fixes one single object, the intention was that clients could implement their own upgrade. i see that we should provide a bit more convenience here) 
- browser view to manually run the upgrade